test/integration/certificate-tests.sh

```diff
@@ -0,0 +1,32 @@
+# AzMiTool Integration tests
+# It requires Bash Testing Framework
+#
+#   this file is part of set of files!
+#
+
+
+#
+# setup variables
+#
+
+identity=$3
+KV_BASENAME=$4
+KV_NA="https://${KV_BASENAME}-no-access.vault.azure.net"
+KV_RO="https://${KV_BASENAME}-r.vault.azure.net"
+
+
+#
+# certificate subcommands testing
+#
+
+test "getcertificate fails on existing but foreign certificate" assert.Fail "azmi getcertificate --certificate ${KV_NA}/certificates/buriedCertificate"
```

Comment:
I am not sure on meaning foreign certificate in this context?

Reply:
Otherwise, all good. Just merge it.

Reply:
Let's say it returns non-zero value if attempt to access KV without access.